### ISSUE TITLE
[BUG] Build fails when using export const { foo: bar } = fooBar;

### ISSUE THREAD

**Issue by grumd** [asapach/babel-plugin-rewire-exports#25]
Sample repository to reproduce the issue:
https://github.com/grumd/babel-plugin-rewire-exports-sample

Steps to reproduce:
1. Clone it, run `npm start`, notice the build fails.
2. Remove `rewire-exports` from `.babelrc`, notice the build is successful.

Problem:
```js
const exportObject = { value: 'Test value' };
export const { value: testValue } = exportObject;
// This is supposed to be imported like this:
// import { testValue } from '...';
```

Stacktrace:
![](https://puu.sh/FPUCQ/530c5c4d92.png)

This type of export syntax isn't supported by this plugin and breaks the build process.

**Comment by grumd** [comment]
Really hope this can get fixed soon because this plugin is a lifesaver :)

**Comment by sergei-startsev** [comment]
@grumd The provided repo doesn't allow to reproduce the issue, there's neither `.babelrc` nor `rewire-exports` in dependencies.

**Comment by grumd** [comment]
@sergei-startsev Very sorry, I forgot to push one commit. Fixed.

**Comment by asapach** [comment]
Should be fixed in v2.2.0

**Comment by grumd** [comment]
@asapach That was incredibly fast, thank you so much!
